test/inductor/test_mps_basic.py

```diff
@@ -85,6 +85,9 @@ def test_cast(self, dtype):
     def test_pointwise_i0(self):
         self.common(torch.special.i0, (torch.rand(128, 128),), check_lowp=False)
 
+    def test_pointwise_i0e(self):
+        self.common(torch.special.i0, (torch.rand(128, 128),), check_lowp=False)
```

Comment:
We need to figure out why `check_lowp` fails, as for element wise ops there shouldn't be any differences...